third-party/realdds/include/realdds/dds-time.h

```diff
@@ -24,6 +24,11 @@ inline dds_time time_from( dds_nsec nanoseconds )
 {
     // see eprosima::fastrtps::rtps::Time_t::from_ns() which includes unneeded fraction calculation
     auto res = std::lldiv( nanoseconds, 1000000000ull );
+    if( res.rem < 0 )
+    {
+        --res.quot;
+        res.rem += 1000000000ull;
```

Comment:
Using this number a lot. Good opportunity to use `constexpr uint64_t nanos_in_second = 1'000'000'000ull`

Reply:
That might be a good idea, but I don't want to introduce this in a header at this point...